turbo.json

```diff
@@ -20,7 +20,7 @@
     "build": {
       "dependsOn": ["^build"],
       "inputs": ["$TURBO_DEFAULT$", ".env*"],
-      "outputs": [".next/**", "!.next/cache/**"]
+      "outputs": [".next/**", "!.next/cache/**", "dist/**"]
```

Comment:
I think this fix random CI fails when `@latitude-data/sdk-js` appears not to be build cc c @csansoon